recipes/libbezier/bld.bat

```diff
@@ -0,0 +1,31 @@
+:: NOTE: This assumes the following environment variables have been set.
+::       - `%SRC_DIR%`
+::       - `%LIBRARY_PREFIX%`
+:: H/T: https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html
+
+mkdir build
+cd build
+
+:: Workaround (somewhat AppVeyor specific) for CMake not wanting sh.exe on
+:: `%PATH%` for MinGW
+set PATH=%PATH:C:\Program Files (x86)\Git\bin;=%
+set PATH=%PATH:C:\Program Files\Git\usr\bin;=%
+
+:: Configure.
+cmake                                              ^
+    -G "MinGW Makefiles"                           ^
+    -DCMAKE_Fortran_COMPILER=gfortran              ^
```

Comment:
1. I don't have any experience with `flang` (on Windows or any other platform) so there is no support for it in the CMake file
1. The build failed when using `flang` as the Fortran compiler because CMake failed to verify it was valid ([this issue][1] indicates it may be an issue with CMake fixed in a newer version)

[1]: https://github.com/trilinos/Trilinos/issues/3694